public/data/developer_assets.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Patrick Obama",
+  "location": "Kenya",
+  "bio": "An aspiring web developer who is an undergrad and recently started exploring open source. Looking for some great opportunities in frontend development. Let's connect!",
+  "avatar": "https://github.com/developer_assets.png",
```

```suggestion
  "avatar": "https://github.com/developer-assets.png",
```

All good but change the avatar url, you have typed wrong url. Check above.